keyboards/crkbd/keymaps/rs/keymap.c

```diff
@@ -0,0 +1,172 @@
+#include QMK_KEYBOARD_H
+#include "oled.h"
+
+enum layers {
+  _QWERTY,
+  _CODE,
+  _FN,
+};
+
+enum custom_keycodes {
+  RGBRST = SAFE_RANGE,
+  CODE,
+  FN,
+  LTGT,
+  LPLT,
+  RPGT,
+  NEQL,
+};
+
+#define KC_ KC_TRNS
+
+#define KC_ESCC MT(MOD_LCTL, KC_ESC)
+#define KC_ENTS MT(MOD_LSFT, KC_ENT)
+#define KC_LTGT LTGT // > or < with shift
+#define KC_LPLT LPLT // ( or < with shift
+#define KC_RPGT RPGT // ) or > with shift
+#define KC_NEQL NEQL // !=
+#define KC_CODE MO(_CODE)
+#define KC_FN   MO(_FN)
+#define KC_RST  RESET
+#define KC_CTRA LCTL(KC_A)
+#define KC_CTRE LCTL(KC_E)
+// Brightness
+#define KC_BRUP KC_PAUS
+#define KC_BRDN KC_SLCK
```

Comment:
These brightness aliases for `KC_PAUS` and `KC_SLCK` are defined in QMK already as `KC_BRMU/BRMD`. But even better, there are media keys that, to my knowledge, work on Windows, macOS *and* Linux: [`KC_BRIU/BRID`](https://docs.qmk.fm/#/keycodes_basic?id=media-keys).

Reply:
Thanks, that's good to know.